src/main/java/bio/terra/workspace/app/configuration/external/BufferServiceConfiguration.java

```diff
@@ -15,13 +15,22 @@
 @ConfigurationProperties(prefix = "workspace.buffer")
 public class BufferServiceConfiguration {
 
+  private boolean enabled = false;
```

Comment:
todo to come back and clean up after cutover?

Reply:
Done.